skimage/segmentation/tests/test_slic.py

```diff
@@ -177,6 +187,7 @@ def test_enforce_connectivity():
     assert_equal(segments_connected_low_max, result_connected)
 
 
+@pytest.mark.filterwarnings("ignore:labels")
```

Comment:
Thanks for ensuring that the warnings stay silenced during tests, but these do highlight the fact that even we all feel that warnings are obtrusive. I would rather there be a more user friendly way to silence these warnings, and suggest it in the warning itself.

Reply:
Warnings are obtrusive, but if we decide to change the behavior of a function, it is important to inform the user ;-)

Reply:
Do you went me to set `start_label` to 0 explicitly in the tests or do we update test results when the behavior changes?
Do we need a test for the validation of error raise if `start_label` is not 0 or 1?